acceptance/bundle/exec/test.toml

```diff
@@ -0,0 +1,2 @@
+Cloud = false
```

Comment:
It might be good to test this with real authentication as provided by our test envs?

Reply:
I don't think there's a real benefit to doing so. The script execution code passes auth credentials along to the downstream process, there's no real benefit in asserting this plumbing works with real credentials.